pkg/framework/fetcher.go

```diff
@@ -21,16 +21,13 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"github.com/m88i/nexus-operator/pkg/logger"
 )
 
 func Fetch(client client.Client, key types.NamespacedName, instance resource.KubernetesResource, kind string) error {
-	log := logger.GetLoggerWithNamespacedName("resource_fetcher", key)
```

Comment:
Why remove the instantiation of a new logger here? The idea is to already have the resource name as a field in the log.

Reply:
we already have a log instance in this package, no need to create a new one. :)

Reply:
Right, but that instance doesn't identify the name of the resource being fetched. I'm ok with not instantiating a new one if we change the actual below into something like:

`log.Info("Attempting to fetch deployed resource", "kind", kind, "namespacedName", key)`

Wdyt?